recipe/bld_pytorch.bat

```diff
@@ -2,6 +2,14 @@
 
 rmdir /s/q build
 
+:: The PyTorch test suite includes some symlinks, which aren't resolved on Windows,
+:: leading to packaging errors.
+:: ATTN! These change and have to be updated manually, often with each release.
+del "%SRC_DIR%\test\_nvfuser\test_python_frontend.py"
```

Comment:
Is this needed when building from the release archive? Or is it just when building from a git checkout?
If the later, you could put this under a condition by setting a env var when rc is not None in meta.yaml

Reply:
no, it's needed when building from the release archive